imageio/plugins/_tifffile.py

```diff
@@ -266,7 +266,7 @@
 *  `python-bioformats <https://github.com/CellProfiler/python-bioformats>`_
 *  `Imread <https://github.com/luispedro/imread>`_
 *  `PyLibTiff <https://github.com/pearu/pylibtiff>`_
-*  `SimpleITK <http://www.simpleitk.org>`_
+*  `ITK <https://www.itk.org>`_
```

Comment:
Note that this file is replaced from time to time with the latest tifffile version.

Reply:
Ah, ok.